config.json

```diff
@@ -216,7 +216,22 @@
           "sequences",
           "strings"
         ]
-      }      
+      },
+      {
+        "slug": "allergies",
+        "name": "Allergies",
+        "uuid": "52b1856c-7570-4d75-aaa9-e1cff3cb6155",
+        "difficulty": 3,
+        "practices": [],
+        "prerequisites": [],
+        "topics" : [
+          "bools",
```

Comment:
Why is this called "bools" unlike all the other words that are real words?  This might be a more systemic problem than just this one PR though.

Reply:
Yeah, I agree. I admit to have copied from Python track. Maybe logicals is better and more consistent with Fortrans logical type.